pkg/boot/multiboot/multiboot.go

```diff
@@ -329,12 +418,12 @@ func (m multiboot) memoryMap() memoryMaps {
 		}
 		ret = append(ret, v)
 	}
+	log.Printf("Memory map: %v", ret)
```

Comment:
do you want all this non-optional noise in the package or could there be a settable Debug() function.

Reply:
leaving this to do for later